test/integration/lerna-publish.test.js

```diff
@@ -57,4 +60,15 @@ describe("lerna publish", () => {
       });
     });
   }));
+
+  test("updates independent versions by npm", () => {
+    return initFixture("PublishCommand/integration").then((cwd) => {
+      return Promise.resolve()
+        .then(() => installInDir(cwd))
+        .then(() => execa("npm", ["run", "lp"], { cwd }))
```

Comment:
We've already got the bootstrap test installing locally and running from an npm script, I don't think we need another one here (just do the `execa(LERNA_BIN...)` part). They take so long to run :P

But yes, I would consider these validation that the same CLI flags work the same way with yargs as they did with meow.